client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -135,7 +140,7 @@ export class CommentDetailAuthor extends Component {
 								{ authorDisplayName }
 							</strong>
 							<span>
-								{ authorUrl }
+								{ withoutHttp( authorUrl ).replace( /\/$/, '' ) }
```

Comment:
I would recommend adding another util method that removes http://, https://, and the last / so we don't need to repeat ourselves. Very easy to unit test too :)